packages/graphiql/test/afterDevServer.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Could we only enable rules of hooks for `jsx` and `tsx` files?

Reply:
I was thinking about this but there is a situation when the custom hook is placed in separate file and doesn't contain JSX elements, so he will have `js`/`ts` extension and not be linted 😒

Reply:
Very good point, then let's leave it as is!